sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -259,6 +259,12 @@ class RapidsExecutorPlugin extends ExecutorPlugin with Logging {
         }
       }
 
+      val reservedShuffleMemory = if (conf.isGPUShuffle) {
+        conf.shuffleTransportMaxReceiveInflightBytes
+      } else {
+        0
+      }
+      GpuMemoryLeaseManager.initialize(reservedShuffleMemory)
       val concurrentGpuTasks = conf.concurrentGpuTasks
       logInfo(s"The number of concurrent GPU tasks allowed is $concurrentGpuTasks")
```

Comment:
nit, could we include in this line the config for the memory lease manager (or ask it to log its own line). It should include the total amount of memory it sees, and optionally add the reserve amount it discounted.